cfgov/paying_for_college/jinja2/paying-for-college/disclosure.html

```diff
@@ -1969,10 +1969,11 @@ <h2 class="step_heading">
                                     eligibility.
                                 </p>
                                 <div class="continue_controls">
-                                    <button class="a-btn a-btn__full-on-xs"
-                                    title="Continue to Step 2" type="button">
-                                    Continue to Step 2
+                                    <button class="a-btn a-btn__full-on-xs a-btn__disabled"
+                                    title="Continue to Step 2" type="button" disabled>
+                                    {{ svg_icon('updating') }} Loading program data...
```

Comment:
You might keep this `Continue to Step 2`, since when it is loading the updating icon will be there and the button will be disabled, so I think that's clear enough. If you do keep this, the `title` attribute should be updated to match the text, before and after loading. I would just simplify it to not update the text via JS though.

Reply:
Changed this!